helm/examples/efs/hpcc-efs/templates/hpcc-efs-volumes.yaml

```diff
@@ -20,16 +20,16 @@ spec:
   resources:
     requests:
       storage: {{ .Values.dataStorage.size }}
-  storageClassName: efs
+  storageClassName: {{ printf "%s" (include "hpcc-efs.fullname" $) }}
```

Comment:
as per prev. comment "%s-sc" or similar.

Reply:
changed in latest submit